### PR TITLE
Trace log line number on error in parsing w3c log file

### PR DESCRIPTION
**Pull request by xavierdecoster**
This further facilitates diagnosing the cause of the error in the source file.

cc @maartenba @skofman1

### CODE REVIEW

File: src/Stats.CollectAzureCdnLogs/Job.cs [hunk 1]
```diff
@@ -291,10 +294,17 @@ private void ProcessLogStream(Stream sourceStream, Stream targetStream, string f
...
         {
-            var parsedEntry = CdnLogEntryParser.ParseLogEntryFromLine(rawLogEntry,
-                e => _logger.LogError(LogEvents.FailedToParseLogFileEntry, e, "Failed to parse W3C log entry in {LogFileName}.", fileName));
+            var parsedEntry = CdnLogEntryParser.ParseLogEntryFromLine(
+                lineNumber,
+                rawLogEntry,
+                (e, line) => _logger.LogError(
+                    LogEvents.FailedToParseLogFileEntry,
+                    e,
+                    "Failed to parse W3C log entry in {LogFileName} at line {LineNumber}.",
```
Comment (maartenba) [inline src/Stats.CollectAzureCdnLogs/Job.cs]: Make it a constant?

File: src/Stats.ImportAzureCdnStatistics/LogFileProcessor.cs [hunk 2]
```diff
@@ -210,13 +210,23 @@ internal class LogFileProcessor
...
-                                e => _logger.LogError(LogEvents.FailedToParseLogFileEntry, e, "Failed to parse W3C log entry in {LogFileName}.", fileName));
+                            lineNumber++;
+
+                            var logEntry = CdnLogEntryParser.ParseLogEntryFromLine(
+                                lineNumber,
+                                rawLogLine,
+                                (e, line) => _logger.LogError(
+                                    LogEvents.FailedToParseLogFileEntry,
+                                    e,
+                                    "Failed to parse W3C log entry in {LogFileName} at line {LineNumber}.",
```
Comment (maartenba) [inline src/Stats.ImportAzureCdnStatistics/LogFileProcessor.cs]: Make it a constant?

**Comment by maartenba** [comment]
LGTM

File: src/Stats.CollectAzureCdnLogs/Job.cs (hunk 1 above)
Comment (xavierdecoster) [inline src/Stats.CollectAzureCdnLogs/Job.cs]: Moved it into resources  in https://github.com/NuGet/NuGet.Jobs/pull/45/commits/2f9651f3fc4ace9c5b47bfbd7d1016dcc5741d4d

File: src/Stats.ImportAzureCdnStatistics/LogFileProcessor.cs (hunk 2 above)
Comment (xavierdecoster) [inline src/Stats.ImportAzureCdnStatistics/LogFileProcessor.cs]: Moved it into resources  in https://github.com/NuGet/NuGet.Jobs/pull/45/commits/2f9651f3fc4ace9c5b47bfbd7d1016dcc5741d4d

**Comment by skofman1** [comment]
:shipit:

**Comment by skofman1** [comment]
LGTM

**Pull request merged**